deeplake/core/vectorstore/vector_search/indra/tql_distance_metrics/order.py

```diff
@@ -0,0 +1,10 @@
+METRIC_TO_ORDER_TYPE = {
+    "l1": "ASC",
+    "l2": "ASC",
+    "cos": "DESC",
+    "max": "ASC",
```

Comment:
did we remove dot?

Reply:
dot is not a distance metric it is vector operation, but there is no clear way how we can understand similarity from a mere dot product. Let's say b has length of 10 and b has length of 1 and c has length of 5. Let's say a is parallel to b and a not parallel but have some angle. if we multiply a and b it will still be 10 while a and c can produce any result up to 50 even if they are not parallel. Was thinking initially to add separate folder with vector ops but I think it doesn't make sense, it is like wrapping every single vector operation with a function, I think user can input this by himself into query if he wants to see the result, but I don't see any practical use cases why we need to support it

Reply:
nope cos is dot divided by magnitude of each vector. So cos uses dot under the hood, but they are not the same